govc/events/command.go

```diff
@@ -45,6 +48,8 @@ func (cmd *events) Register(ctx context.Context, f *flag.FlagSet) {
 	cmd.DatacenterFlag.Register(ctx, f)
 
 	f.IntVar(&cmd.Max, "n", 25, "Output the last N events")
+	f.BoolVar(&cmd.Tail, "t", false, "Tail events")
```

Comment:
I still like -f better, since 'events' is already similar to the 'tail' command.  We use the same -n option as tail.  Thoughts?